### PR TITLE
gatk-sv: summarise qc, batching & SV tools

### PR DESCRIPTION
**Pull request by pdiakumis**
Some short notes from the gnomad-sv paper

### CODE REVIEW

File: gatk-sv/README.md [hunk 1]
```diff
@@ -1,14 +1,116 @@
...
 
-## Module00a
+- [GATK-SV](#gatk-sv)
+  - [Module00 - Preprocessing](#module00---preprocessing)
+    - [Ploidy estimation & sex inference](#ploidy-estimation--sex-inference)
+    - [Sequencing dosage bias scoring](#sequencing-dosage-bias-scoring)
+    - [Sample QC](#sample-qc)
+    - [Sample batching](#sample-batching)
+    - [Execution of SV discovery algorithms](#execution-of-sv-discovery-algorithms)
+    - [Module00a](#module00a)
```
Comment (KatalinaBobowik) [inline gatk-sv/README.md:10]: Maybe you could add what module00a is (SV evidence collection, etc)

File: gatk-sv/README.md [hunk 2]
```diff
@@ -1,14 +1,116 @@
...
+    - [Module00b](#module00b)
```
Comment (KatalinaBobowik) [inline gatk-sv/README.md:11]: same for these ones

File: gatk-sv/README.md [hunk 3]
```diff
@@ -1,14 +1,116 @@
...
+  - maximises sensitivity for small and balanced SVs
+  - v0.7.7 in single-sample mode for del/dup/ins/inv
+- **MELT**:
+  - captures mobile element insertions (MEIs) with high sensitivity
+  - v2.0.5 in single-sample mode
+- **cn.MOPS**:
+
+  - flexible RD-based algo for cohorts with high sensitivity for rare CNVs.
+  - custom implementation of v1.20.1 in ~100 sample batches
+  - compose coverage matrixes across all samples at 300bp and 1000bp resolution,
```
Comment (KatalinaBobowik) [inline gatk-sv/README.md:88]: matrices instead of matrixes

File: gatk-sv/README.md [hunk 4]
```diff
@@ -1,14 +1,116 @@
...
+
+First collect required data & metadata, then perform the next steps.
+
+### Ploidy estimation & sex inference
+
+- read depth in 1Mb bins:
+  - exclude bins with no coverage in >5% samples
+  - normalise sample coverage by:
+    `tot cov per bin / median across all autosomal bins`
+- per-chrom ploidy
```
Comment (KatalinaBobowik) [inline gatk-sv/README.md:25]: I would spell out chromosome, but that's just me

File: gatk-sv/README.md [hunk 5]
```diff
@@ -1,14 +1,116 @@
...
+  - 2x median normalised coverage per bin
+- sex inference:
+
+  - round sex chrom ploidy to nearest integer copy state
+
+### Sequencing dosage bias scoring
+
+- confounded CNV calling due to highly non-uniform coverage
```
Comment (KatalinaBobowik) [inline gatk-sv/README.md:33]:
```suggestion
- confounded copy number variation (CNV) calling due to highly non-uniform coverage
```

File: gatk-sv/README.md (hunk 4 above)
Comment (KatalinaBobowik) [inline gatk-sv/README.md:25]:
```suggestion
- per-chromosome ploidy
```

File: gatk-sv/README.md [hunk 6]
```diff
@@ -1,14 +1,116 @@
...
+
+- read depth in 1Mb bins:
+  - exclude bins with no coverage in >5% samples
+  - normalise sample coverage by:
+    `tot cov per bin / median across all autosomal bins`
+- per-chrom ploidy
+  - 2x median normalised coverage per bin
+- sex inference:
+
+  - round sex chrom ploidy to nearest integer copy state
```
Comment (KatalinaBobowik) [inline gatk-sv/README.md:29]:
```suggestion
  - round sex chromosome ploidy to nearest integer copy state
```

File: gatk-sv/README.md [hunk 7]
```diff
@@ -1,14 +1,116 @@
...
+
+### Sequencing dosage bias scoring
+
+- confounded CNV calling due to highly non-uniform coverage
+- antipodal between PCR- and PCR+ protocols
+- WGD package controls for dosage bias:
```
Comment (KatalinaBobowik) [inline gatk-sv/README.md:35]:
```suggestion
- Whole-Genome Dosage (WGD) package controls for dosage bias:
```

File: gatk-sv/README.md [hunk 8]
```diff
@@ -1,14 +1,116 @@
...
+  - theta metric summarises directionality and magnitude of bias per sample
+
+### Sample QC
+
+- Median 100bp bin coverage
+- Dosage bias score theta: measurement of coverage uniformity with WGD.
+- Autosomal ploidy spread: diff between highest/lowest ploidies for any two
```
Comment (KatalinaBobowik) [inline gatk-sv/README.md:42]:
```suggestion
- Autosomal ploidy spread: difference between highest/lowest ploidies for any two
```

File: gatk-sv/README.md [hunk 9]
```diff
@@ -1,14 +1,116 @@
...
+- Z-score of outlier 1Mb bins: median absolute Z-score of number of 1Mb bins per
+  chromosome with normalised CN estimates < 1.5 or >2.5 Z-scores were calculated
+  separately for PCR+ and PCR- samples.
+- Chimera rate: percentage of chimeric read pairs in total read pairs.
+- Pairwise alignment rate: percentage of read pairs where both reads aligned
+  successfully.
+- Library contamination: max value of adapter or est. sample contamination
+  fraction.
+- Read length: mean read length
+- Ambiguous sex genotypes: normalised CN est. for chrX & chrY; ambiguous if
```
Comment (KatalinaBobowik) [inline gatk-sv/README.md:53]: normalised copy number estimate?

File: gatk-sv/README.md (hunk 3 above)
Comment (KatalinaBobowik) [inline gatk-sv/README.md:88]:
```suggestion
  - compose coverage matrices across all samples at 300bp and 1000bp resolution,
```

File: gatk-sv/README.md [hunk 10]
```diff
@@ -1,14 +1,116 @@
...
+- Dosage bias score theta: measurement of coverage uniformity with WGD.
+- Autosomal ploidy spread: diff between highest/lowest ploidies for any two
+  autosomes.
+- Z-score of outlier 1Mb bins: median absolute Z-score of number of 1Mb bins per
+  chromosome with normalised CN estimates < 1.5 or >2.5 Z-scores were calculated
+  separately for PCR+ and PCR- samples.
+- Chimera rate: percentage of chimeric read pairs in total read pairs.
+- Pairwise alignment rate: percentage of read pairs where both reads aligned
+  successfully.
+- Library contamination: max value of adapter or est. sample contamination
```
Comment (KatalinaBobowik) [inline gatk-sv/README.md:50]: I don't understand this line

**Review by KatalinaBobowik — APPROVED** [review]
Looks great!

File: gatk-sv/README.md (hunk 10 above)
Comment (pdiakumis) [reply gatk-sv/README.md:50]:
```suggestion
- Library contamination: max value of adapter contamination or estimated sample contamination
```

File: gatk-sv/README.md (hunk 9 above)
Comment (pdiakumis) [inline gatk-sv/README.md:53]:
```suggestion
- Ambiguous sex genotypes: normalised copy number estimated for chrX & chrY; ambiguous if
```

**Pull request merged**